spyglass/server.py

```diff
@@ -26,34 +26,42 @@ class StreamingServer(socketserver.ThreadingMixIn, server.HTTPServer):
 
 def run_server(bind_address, port, output):
     class StreamingHandler(server.BaseHTTPRequestHandler):
+        streamURL = '/stream'
+        snapshotURL = '/snapshot'
```

Comment:
AFAIK the Python convention would be
```suggestion
        stream_url = '/stream'
        snapshot_url = '/snapshot'
```